### DISCUSSION TITLE
SSD1306 on Pi Zero ??

### DISCUSSION THREAD

**Discussion by scuzzphut** [probonopd/MiniDexed#498] [Q&A] [answered]
Hi All,

I'm trying to get Dexed going on a Pi Zero , using Kevin's board here :   

https://diyelectromusic.wordpress.com/2022/08/16/minidexed-raspberry-pi-io-board-part-2/

Everything is working except the SSD1306 display.  I have double checked the connections and the software setup and tried multiple SSD1306 displays (admittedly all from the same batch) but no joy.

Is it possible that the SSD1306 is not supported on the Pi Zero ?   Anyone have this working ? 

Many thanks !

**Comment by scuzzphut** [comment, marked as answer]
Never mind - got it working. 

I had LCDEnabled=0 , inexplicably. 

Leaving this here.  

Device is working and is an EstarDyn 0.96 inch IIC SPI Serial 7/4Pin White/Blue/Yellow Blue/Yellow OLED Display Module SSD1306 12864 LCD Screen Board for Arduino

https://www.aliexpress.com/item/1005001621806398.html?spm=a2g0s.12269583.0.0.b74b7218PlVAek

**Reply by probonopd** [reply]
Great that it is working for you @scuzzphut :+1:
